app/assets/stylesheets/views/_brexit_checker_results_page.scss

```diff
@@ -14,12 +36,40 @@
     background-color: govuk-colour('grey-4');
   }
 
-  .stay-updated__heading,
+  .brexit-checker-actions__group {
+    padding-top: govuk-spacing(6);
+    border-bottom: 1px solid $govuk-border-colour;
+    @include govuk-media-query($until: tablet) {
+      padding-top: govuk-spacing(4);
+    }
+  }
+  .brexit-checker-actions__group:last-child {
+    border-bottom: none;
+  }
+
+  .stay-updated__heading {
+    margin-top: govuk-spacing(6);
+  }
+
+  .no_results {
+    margin-top: govuk-spacing(6);
+    padding-top: govuk-spacing(6);
+    border-top: 3px solid $govuk-brand-colour;
+    border-bottom: 3px solid $govuk-brand-colour;
+  }
+
+  .no-results-warning {
+    background-color:#f8f8f8;
```

Comment:
Yeah i wasn't sure about this, the current page uses `f8f8f8` in a couple of places, I'd tried replacing it with `govuk-colour("light-grey")` | `#f3f2f1`, which is nearly but not quite the same.

Perhaps I should replace in both places?

Reply:
Probably best to be consistent and replace everywhere. This is probably due to a mismatch in colours because we're kind of half moved from `govuk_frontend_toolkit` to `govuk-frontend` and they have slightly different colours. Light grey should be fine though.

Reply:
cool, will amend